app/views/post/list.html.erb

```diff
@@ -0,0 +1 @@
+<h3>Here is a list of posts for given user</h3>
```

Comment:
- [ ] Kindly rename the folder containing this file from `post` to `posts` to match the controller name.

- [ ] This file name should be `index.html.erb`. Kindly rename the file.